eclipse-collections-code-generator/src/main/resources/test/list/mutable/synchronizedPrimitiveListTest.stg

```diff
@@ -51,6 +55,20 @@ public class Synchronized<name>ListTest extends Abstract<name>ListTestCase
         Assert.assertSame(list, list.asSynchronized());
         Assert.assertEquals(list, list.asSynchronized());
     }
+<if(primitive.specializedStream)>
+
+    @Test
+    public void stream()
+    {
+        Assert.assertEquals(Arrays.asList(<["1", "2", "3"]:(literal.(type))(); separator=", ">), <name>ArrayList.newListWith(<["1", "2", "3"]:(literal.(type))(); separator=", ">).asSynchronized().primitveStream().boxed().collect(Collectors.toList()));
+    }
+
+    @Test
+    public void parallelStream()
+    {
+        Assert.assertEquals(Arrays.asList(<["1", "2", "3"]:(literal.(type))(); separator=", ">), <name>ArrayList.newListWith(<["1", "2", "3"]:(literal.(type))(); separator=", ">).asSynchronized().primitveParallelStream().boxed().collect(Collectors.toList()));
+    }
```

Comment:
These probably should only be in the superclass. Here and in the unmodifiable test, we have assertions that aren't related to being synchronized or unmodifiable.